compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -309,7 +309,7 @@ impl fmt::Display for UndefinedBehaviorInfo<'_> {
                 allocation_size.bytes()
             ),
             DanglingIntPointer(_, CheckInAllocMsg::NullPointerTest) => {
-                write!(f, "NULL pointer is not allowed for this operation")
+                write!(f, "null pointer is not allowed for this operation")
```

Comment:
Hm, looks like we don't have any test covering this output... I think `CheckInAllocMsg::NullPointerTest` might just be a dead enum variant.
@hyd-dev any chance you could prepare a PR to remove it?

Reply:
Sure, I'll have a try!

Reply:
Created #84903 to remove it.